src/lib/generate-starterpack.ts

```diff
@@ -111,7 +109,20 @@ export async function generateStarterpack({
       )
     ).uri
   } catch (e: unknown) {
-    logger.error('Failed to generate starter pack', {error: e})
+    logger.error('Failed to generate starter pack', {safeMessage: `${e}`})
```

Comment:
Doesn't matter but FYI you actually don't need to coerce here, we check for `instanceof Error` in the breadcrumbs handling and call `Error.toString` https://github.com/bluesky-social/social-app/blob/main/src/logger/index.ts#L97

Reply:
ah wasn't sure if we did it for each value or not, thanks.